core/arch/arm/dts/stm32mp157c-ed1.dts

```diff
@@ -90,10 +90,14 @@
 		regulator-type = "voltage";
 		regulator-always-on;
 
+//		gpios = <&gpiof 14 (GPIO_STM32_NON_SECURE |GPIO_ACTIVE_HIGH)>;
 		gpios = <&gpiof 14 GPIO_ACTIVE_HIGH>;
+
 		gpios-states = <0>;
 		states = <1800000 0x1>,
 			 <2900000 0x0>;
+		/* SD switch is not used by OP-TEE */
```

Comment:
I think I can also remove this inline comment. The Git log should be enough.

Reply:
Agreed.